cpmonitor/static/js/mstr.js

```diff
@@ -0,0 +1,72 @@
+(async () => {
+  const response = await fetch("/api/mstr/" + LZM.city.municipalityKey);
+
+  const el = document.querySelector("#kpiPV .canvas");
```

Comment:
maybe name it something more meaningful like kpiPvCanvas

Reply:
renamed it to canvasEl